docs/tutorials/dronecan_firmware_upgrade.rst

```diff
@@ -0,0 +1,130 @@
+
+Here, you can view other devices detected on the bus, configure devices, and importantly here, trigger firmware updates. In order to interact with other DroneCAN nodes on the bus, 
+the USB-to-CAN node needs its own unique node ID. To set your device's node ID, find *Set local node ID* in the top left, pick a node ID unique from all other devices on the 
+bus (the default 127 is unique from the default Vertiq node ID of 99), and click the check mark.
+
+Now, connect your module to your USB-to-CAN device, and power it on. You should see it appear in the list of *Online nodes*:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_seen.png
+
+Double click the module's node to find its DroneCAN node properties
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_properties.png
+    :scale: 80%
+
+From here, you can view the module's parameters by selecting *Fetch All*, you can reboot your module with *Restart*, or you can start a firmware update with *Update Firmware*.
+
+.. _dronecan_upgrade_instructions:
+
+Updating Firmware with the DroneCAN GUI
+============================================
+Now that you have a basic understanding of the DroneCAN GUI's layout and interactions, we can move on to updating firmware through the GUI tool. First, you must download the correct firmware for your 
+module from `our website <https://www.vertiq.co/>`_. Take care to ensure that the firmware you download is meant for your module (size, Kv, firmware style, etc.). 
+Failure to do so can result in severe damage to your module. Inside of the downloaded .zip archive, you should find a file called combined.bin as well as a file called app.bin. 
+
+.. note::
+    If your zip archive does not include a file named app.bin, you cannot update your module's firmware via DroneCAN. Future firmware releases will include app.bin files for all DroneCAN capable modules.
+
+Now, extract the archive's files to a location of your choosing. You will need to find this location again in the next step. In this case, the files have been extracted 
+to a folder called DroneCAN Upgrade Demo inside of the Documents folder.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/extracted_files.png
+
+
+Navigate back to the DroneCAN GUI, and open your module's *Node Properties* window. Select *Update Firmware*.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/update_firmware_highlight.png
+    :scale: 70%
+
+This will open a file dialogue. Navigate to the location of your extracted archive, and inside you will see the app.bin and combined.bin files.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/app_and_upgrade_files.png
+
+**Select app.bin**, and click Open.
+
+.. warning::
+    It is crucial that you do not select combined.bin. Attempting to flash a combined application through the DroneCAN firmware upgrader will overwrite important module data, and will render your module unusable.
```

Comment:
Would the module be at least recoverable? I think it possibly would be, since the app section should look weird to the boot if this happens, but maybe double check that. If so maybe that last bit could be slightly less scary and a bit more accurate if it said "...and will recover you to recover your module through the Control Center"

Reply:
Just tried it, and it is not recoverable if you flash combined with dronecan